src/UI/Shared/FormatHelpers.js

```diff
@@ -3,16 +3,19 @@ var filesize = require('filesize');
 var UiSettings = require('./UiSettingsModel');
 
 module.exports = {
-    bytes : function(sourceSize) {
+    bytes : function(sourceSize, sourceRounding) {
         var size = Number(sourceSize);
-
+        var rounding = Number(sourceRounding||1);
         if (isNaN(size)) {
```

Comment:
have added line break above if here and in the other place. Do you have a wiki page with conventions? I saw the one about the tabs but not line breaks before ifs